lib/WorkflowRnaseq.groovy

```diff
@@ -86,6 +86,9 @@ class WorkflowRnaseq {
             if (params.rsem_index && params.star_index) {
                 rsemStarIndexWarn(log)
             }
+            if (params.star_extra_alignment_args) {
```

Comment:
```suggestion
            if (params.aligner  == 'star_rsem' && params.extra_star_align_args) {
```

Reply:
Isn't the `params.aligner == 'star_rsem' &&` part redundant? But I don't mind having it.

And yes, I can see how using the correct variable name is beneficial...